kickstart/PXELinux.erb

```diff
@@ -20,19 +20,40 @@ oses:
     options << "modprobe.blacklist=" + @host.params['blacklist'].gsub(' ', '')
   end
   options = options.join(' ')
+  # handle non-DHCP environments (e.g. bootdisk)
+  subnet = @host.subnet
+  if subnet.respond_to?(:dhcp_boot_mode?)
```

Comment:
`subnet` will always respond to `dhcp_boot_mode?` on the develop branch, no conditional required